OracleDatabase/SingleInstance/dockerfiles/18.4.0/Dockerfile.xe

```diff
@@ -58,9 +58,9 @@ RUN chmod ug+x $INSTALL_DIR/*.sh && \
     sync && \
     $INSTALL_DIR/$CHECK_SPACE_FILE && \
     cd $INSTALL_DIR && \
-    yum -y install openssl oracle-database-preinstall-18c && \
+    yum -y install expect openssl oracle-database-preinstall-18c && \
     sed -i -e 's/\(oracle\s\+hard\s\+nofile\)/# \1/' /etc/security/limits.d/oracle-database-preinstall-18c.conf && \
-    yum -y localinstall $INSTALL_FILE_1 && \
+    unbuffer yum -y localinstall $INSTALL_FILE_1 && \
```

Comment:
`localinstall` isn't needed.

Reply:
If `unbuffer` is only used for the installation itself and not required for the runtime of the container, we should remove the `except` package again at the end (before cleaning the yum caches).

Reply:
The `expect` package adds 5MB to the final image. It's hardly noticeable if it remains and the `unbuffer` tool would be useful for any downstream image that uses `yum`.

Reply:
And? If it's not needed afterwards anymore, why keep it in the image?

Reply:
Removed package `expect` before cleaning yum cache.